pyccel/errors/errors.py

```diff
@@ -325,10 +325,7 @@ def report(self,
         if symbol is not None:
             if isinstance(symbol, ast.AST):
                 ast_node = symbol
-                if sys.version_info < (3, 9):
-                    symbol = ast.dump(ast_node)
-                else:
-                    symbol = ast.unparse(ast_node) # pylint: disable=no-member
+                symbol = ast.unparse(ast_node) # pylint: disable=no-member
```

Comment:
We should not need the Pylint command `# pylint: disable=no-member` any longer, right?

Reply:
Ah I think you are right!

Reply:
Fixed [c1f6db5](https://github.com/pyccel/pyccel/pull/2062/commits/c1f6db557c2b03ff0f38e83b95d988dfb02d4e2b)